docs/running-on-mesos.md

```diff
@@ -228,7 +228,11 @@ in your [SparkConf](configuration.html#spark-properties).
 The Docker image used must have an appropriate version of Spark already part of the image, or you can
 have Mesos download Spark via the usual methods.
 
-Requires Mesos version 0.20.1 or later.
+Note that by default Mesos agents will not pull the image if it already exists on the agent. If you use mutable image
+tags you can set `spark.mesos.executor.docker.forcePullImage` to `true` in order to force the agent to always pull the
+image before running the executor.
+
+Requires Mesos version 0.22.2 or later.
```

Comment:
Docker image support, generally, will still work in 0.20.1.  It's just the forcePullImage feature that will require 0.22.2.